custom_components/dahua/__init__.py

```diff
@@ -549,6 +552,10 @@ def is_flood_light(self) -> bool:
         m = self.model.upper()
         return m.startswith("ASH26") or "L26N" in m or "L46N" in m or m.startswith("V261LC") or m.startswith("W452ASD")
 
+    def supports_floodlightmode(self) -> bool:
+        """ Returns true if this camera supports floodlight mode """
+        return "W452ASD" in self.model.upper() or "L46N" in self.model.upper()
```

Comment:
I think one of my cams is a different model but supports the floodlight. Once I get home I'll try to check...

Reply:
ASH26 flood light is still broken in 0.9.54. Anyway we can get this merged on the next release?